src/chrome/local.ts

```diff
@@ -35,22 +35,24 @@ export default class LocalChrome implements Chrome {
   }
 
   private async startChrome(): Promise<Client> {
+    const { port } = this.options.cdp
     this.chromeInstance = await launch({
       logLevel: this.options.debug ? 'info' : 'silent',
-      port: this.options.cdp.port,
+      port,
     })
     const target = await CDP.New({
-      port: this.chromeInstance.port,
```

Comment:
I don't think accessing port via `this.options.cdp` is the right way of doing this.

`chrome-launch` takes 0 for the port value and will generate a random port number for us. This port number can correctly be accessed via `this.chromeInstance` but not via `this.options.cdp`.

Reply:
Any suggestions for accessing this.chromeInstance here in util.ts? https://github.com/anttiviljami/chromeless/blob/475dd3e9988ff23ca2705e1a45c17b7bac1521e4/src/util.ts#L27

Reply:
Should we just make `chromeInstance` a public property for the client?